segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListenerTest.java

```diff
@@ -195,4 +198,29 @@ public void testCreateRequestProcessor() {
                 mock(StreamSegmentStore.class), mock(TableStore.class), NoOpScheduledExecutor.get());
         Assert.assertTrue(listener.createRequestProcessor(new TrackedConnection(new ServerConnectionInboundHandler())) instanceof AppendProcessor);
     }
+
+    // Test the health status created with pravega listener.
+    @Test
+    public void testHealth() {
+        int port = TestUtils.getAvailableListenPort();
+        PravegaConnectionListener listener = new PravegaConnectionListener(false, port,
```

Comment:
Does this class implement `AutoCloseable` or does it have a `close` or `shutdown` method? If so, add the `@Cleanup` to it to ensure that any resources it acquired during its lifetime are properly cleaned up when we get rid of it.